### PR TITLE
Fixing dead links in TOC and other fixes

### PR DESCRIPTION
**Pull request by ljzhong**
Same 3 errors:

Package & Publish
"Get your app approved"
TOC doesn't slide open

Known issues:
second "Nuget Package Manager" should be Plugins Installation

under tutorials and samples 2 links were broken

### CODE REVIEW

File: dps-web-experience-sample/Views/Shared/_DocsNav.cshtml [hunk 1]
```diff
@@ -124,9 +124,9 @@
 
                 <li id="tutorial-ionic"><a href="~/en-us/docs/tutorial-ionic/">Get started with Ionic</a></li>
 
-                <li id="O365_Files"><a href="~/en-us/docs/O365_Files/">O365 Files API</a></li>
+                <li id="O365_Files"><a href="~/en-us/docs/o365_Files/">O365 Files API</a></li>
```
Comment (rakatyal) [inline dps-web-experience-sample/Views/Shared/_DocsNav.cshtml:127]: o365_files

File: dps-web-experience-sample/Views/Shared/_DocsNav.cshtml [hunk 2]
```diff
@@ -124,9 +124,9 @@
...
 
-                <li id="O365_Ionic"><a href="~/en-us/docs/O365_Ionic/">O365 Outlook and Ionic</a></li>
+                <li id="O365_Ionic"><a href="~/en-us/docs/o365_Ionic/">O365 Outlook and Ionic</a></li>
```
Comment (rakatyal) [inline dps-web-experience-sample/Views/Shared/_DocsNav.cshtml:129]: o365_ionic

Comment (rakatyal) [inline dps-web-experience-sample/Views/Shared/_DocsNav.cshtml:129]: Please make the same changes for the small_nav section as well. Also you will have to update index.js file.

Comment (ljzhong) [inline dps-web-experience-sample/Views/Shared/_DocsNav.cshtml:129]: easier said than done...

**Pull request closed without merging**